packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt

```diff
@@ -387,37 +386,13 @@ public class AuthCognito : FlutterPlugin, ActivityAware, MethodCallHandler, Plug
 
   private fun onGetCurrentUser(@NonNull flutterResult: Result) {
     try {
-      /* 
-        Because Android does not preserve the user after session expiration but iOS does so,
-        we need to use the mobileclient to get the old username to enforce platform parity.
-      */
-      Amplify.Auth.fetchAuthSession(
-        { result ->
-          val cognitoAuthSession = result as AWSCognitoAuthSession
-          when (cognitoAuthSession.userSub.type) {
-            // If the user sub accessor successful the user is present and the session should be valid
-            AuthSessionResult.Type.SUCCESS -> {
-              val awsMobileClient = AWSMobileClient.getInstance()
-              val username = awsMobileClient.username
-              prepareUserResult(flutterResult, AuthUser(cognitoAuthSession.userSub.toString(), username));
-            }
-            // If the user sub accessor failed, check the signIn state
-            AuthSessionResult.Type.FAILURE -> {
-              // if signedIn flag still true, we know session is expired so get the old user
-              if (result.isSignedIn) {
-                val awsMobileClient = AWSMobileClient.getInstance()
-                val username = awsMobileClient.username
-                val userid = awsMobileClient.userSub
-                prepareUserResult(flutterResult, AuthUser(userid, username));
-              // if signIn flag is false, we assume user is signed out so throw exception
-              } else {
-                errorHandler.handleAuthError(flutterResult, AuthException.SignedOutException())
-              }
-            }
-          }
-        },
-        { error -> errorHandler.handleAuthError(flutterResult, error) }
-      )
+      var user: AuthUser? = Amplify.Auth.currentUser;
```

Comment:
Is this overwriting the changes made in release-candidate?

Reply:
Weird... this wasn't even marked as a conflict.

Reply:
@dnys1 Yeah  - I have no idea why, none of these were marked as conflicts. I think I have this manually resolved now.